client-course-schedulizer/src/components/Tabs/FacultyLoads/UpdateSectionModalPagination/UpdateSectionModalPagination.tsx

```diff
@@ -0,0 +1,87 @@
+import { AddSectionPopover, ModalPagination } from "components";
+import React, {
+  ChangeEvent,
+  forwardRef,
+  Ref,
+  useContext,
+  useImperativeHandle,
+  useState,
+} from "react";
+import {
+  CourseSectionMeeting,
+  CourseSectionMeetingTermSections,
+  findSection,
+  Term,
+  UpdateSectionModalPaginationRef,
+} from "utilities";
+import { AppContext } from "utilities/contexts";
+import "./UpdateSectionModalPagination.scss";
+
+interface TermSections {
+  sections: string[];
+  term: Term;
+}
+
+export const UpdateSectionModalPagination = forwardRef(
+  (props, ref: Ref<UpdateSectionModalPaginationRef>) => {
+    const {
+      appState: { schedule },
+    } = useContext(AppContext);
+
+    const [open, setOpen] = useState(false);
+    const [modalValues, setModalValues] = useState<CourseSectionMeeting>();
+    const [sectionNames, setSectionNames] = useState<TermSections>({
+      sections: [],
+      term: Term.Fall,
+    });
+    const [page, setPage] = useState(1);
```

Comment:
Possibly set this to use the class you clicked on instead of going to the first page always.